modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -479,3 +481,70 @@ test('can import custom object entries at instance level with or without permiss
 		})
 	);
 });
+
+test('can see corresponding elements in instance and site level', async ({
+	apiHelpers,
+	companyExportImportPage,
+	exportImportPage,
+}) => {
+	const objectActionApiClient =
+		await apiHelpers.buildRestClient(ObjectDefinitionApi);
+
+	const {body: objectDefinition} =
+		await objectActionApiClient.postObjectDefinition({
+			active: true,
+			externalReferenceCode: 'test',
+			label: {
+				en_US: 'Test',
+			},
+			name: 'Test',
+			objectFields: [
+				{
+					DBType: ObjectField.DBTypeEnum.String,
+					businessType: ObjectField.BusinessTypeEnum.Text,
+					indexed: true,
+					indexedAsKeyword: true,
+					label: {
+						en_US: 'Name',
+					},
+					name: 'name',
+					required: true,
+				},
+			],
+			pluralLabel: {
+				en_US: 'Tests',
+			},
+			portlet: true,
+			scope: 'company',
+			status: {
+				code: 0,
+			},
+		});
+
+	apiHelpers.data.push({id: objectDefinition.id, type: 'objectDefinition'});
+
+	const exportFilePath =
+		await companyExportImportPage.export('Tests 1 Items');
+
+	await exportImportPage.page.goto('/');
+
+	await exportImportPage.goToImportOptions(exportFilePath);
+
+	await expect(
+		companyExportImportPage.page.getByText('Comments, Ratings')
```

Comment:
This is weird, as you are asserting to be visible a page of the companyExportImportPage, is this correct?

Reply:
What I'm doing here is to check that this is not visible in the page of `companyExportImportPage`
![image](https://github.com/user-attachments/assets/a5e104e2-86a2-4874-b824-b2f83a325341)